### PR TITLE
Change test names for consistency

### LINKED ISSUE #173: Change tests names for consistency
Currently we have several tests running, change their names to be consistent.

### PR DESCRIPTION
**Pull request by georgi-l95**
Signed-off-by: georgi-l95 <glazarov95@gmail.com>

**Description**:

- Change test names for consistency.
- Bump image versions and take advantage of the new faster mode for local-node
- Remove some unused imports
- Increase fee for token creation ( the current one is enough for local-node, but not for testnet/previewnet )

**Related issue(s)**:

Fixes #173 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -61,3 +61,9 @@ jobs:
     uses: ./.github/workflows/test-workflow.yml
     with:
       testfilter: HIP583
+
+  Multicall:
+    name: Multicall Test Suite
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      testfilter: Multicall
```
Comment (Nana-EC) [inline .github/workflows/tests.yml:69]: nit: new line

File: test/hts-precompile/utils.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 const { ethers } = require("hardhat");
 const { expect } = require("chai");
-const { LocalProvider, Wallet, AccountId, Client, AccountInfoQuery } = require("@hashgraph/sdk");
+const { AccountId, Client, AccountInfoQuery } = require("@hashgraph/sdk");
 
 class Utils {
   //createTokenCost is cost for creating the token, which is passed to the precompile. This is equivalent of 10 and 20hbars, any excess hbars are refunded.
-  static createTokenCost = "10000000000000000000";
-  static createTokenCustomFeesCost = "20000000000000000000";
+  static createTokenCost = "40000000000000000000";
```
Comment (Nana-EC) [inline test/hts-precompile/utils.js:7]: Update comment above

**Review by Nana-EC — COMMENTED** [review]
Looking good. Minor nits

File: .github/workflows/tests.yml (hunk 1 above)
Comment (georgi-l95) [reply .github/workflows/tests.yml]: Added.

File: test/hts-precompile/utils.js (hunk 2 above)
Comment (georgi-l95) [reply test/hts-precompile/utils.js]: Updated.

**Review by Nana-EC — APPROVED** [review]

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**